Gemfile.lock

```diff
@@ -293,14 +293,14 @@ GEM
       ruby-openid (>= 2.1.8)
     rack-protection (2.0.3)
       rack
-    rack-test (1.0.0)
+    rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.0.4)
       loofah (~> 2.2, >= 2.2.2)
-    rails_multisite (2.0.5)
+    rails_multisite (2.0.4)
```

Comment:
Why is this one rolling back multisite?

Reply:
Good question - let me look into that

Reply:
Ahh, it's because I've branched off a very old version. Let me fix that.

Reply:
Although also, it looks like 2.0.5 might have been yanked: https://rubygems.org/gems/rails_multisite

Reply:
Yeah, it was yanked: https://rubygems.org/gems/rails_multisite/versions/2.0.5